### ISSUE TITLE
Wrap Items

### ISSUE THREAD

**Issue by mbcoder17** [uacaps/PageMenu#64]
Is there a way to wrap the items in the menu around to the other side? For example, if I have 3 items in my menu, one of them will always be in the center, while the remaining two will each be on one side of the one in the middle. So, when I swipe to the left, the one on the right will move to the center, and the one on the left will appear on the right side.

**Comment by fahlout** [comment]
This would be a nice feature to have, but is currently available in this library, sorry

**Comment by mbcoder17** [comment]
No problem, I was able to build my own using a similar library found here: https://github.com/malcommac/DMCircularScrollView

**Comment by serhatsezer** [comment]
Yes it would be really nice feature! Please add this feature in your feature list :+1:
